src/features/myAccounts/components/AccountItems/MultiAccountItem.tsx

```diff
@@ -179,13 +180,33 @@ const MultiAccountItem = ({ onLinkClick, multiSafeAccountItem }: MultiAccountIte
         dispatch(pinSafe({ chainId: safe.chainId, address: safe.address }))
       }
     }
+
+    dispatch(
+      showNotification({
+        title: 'Pinned Multichain Safe',
```

Comment:
```suggestion
        title: 'Pinned multi-chain Safe',
```

Reply:
Done! I also changed the notification message to either show the safe name or the shortened address as per @TanyaEfremova suggestion